gatsby/src/pages/about/{BnfAboutSection.slug}.tsx

```diff
@@ -14,26 +15,118 @@ export type AboutSectionPageProps = {
+		);
+		metas.set(
+			"/about/bnf-staff/",
+			`View the content director, editorial staff and support staff responsible for producing the ${siteTitleShort}.`
+		);
+		metas.set(
+			"/about/joint-formulary-committee/",
+			`The Joint Formulary Committee (JFC) is responsible for the content of the ${siteTitleShort}. The JFC includes pharmacy, medical, nursing, and lay representatives.`
+		);
+		metas.set(
+			"/about/dental-advisory-group/",
+			`The Dental Advisory Group oversees preparation of advice on the drug management of dental and oral conditions.`
+		);
+		metas.set(
+			"/about/nurse-prescribers-advisory-group/",
+			`The Nurse Prescribers' Advisory Group oversees the list of drugs approved for inclusion in the Nurse Prescribers' Formulary.`
+		);
+		metas.set(
+			"/about/how-bnf-publications-are-constructed/",
+			`Read more about the editorial and production processes behind the construction of the ${siteTitleShort} and the assessment of evidence used to make recommendations.`
+		);
+		metas.set(
+			"/about/how-to-use-bnf-publications-online/",
+			`View the how-to guide for using the online version of the ${siteTitleShort}, as found in appendix 1 of the printed edition.`
+		);
+		metas.set(
+			"/about/frequently-asked-questions-for-the-bnf-and-bnf-for-children-bnfc-general/",
+			`Get answers to frequently asked questions about the British National Formulary, including how to cite, or reference, the different versions of the BNF and BNFc.`
+		);
+		metas.set(
+			"/about/frequently-asked-questions-clinical/",
+			`View the answers to commonly asked questions that may come up when using the ${siteTitleShort} information and guidance on prescribing.`
+		);
+		metas.set(
+			"/about/approximate-conversions-and-units/",
+			`Conversions and units tables including growth chart with average weight and height, by gender and age (neonate, child and adult).`
+		);
+		metas.set(
+			"/about/abbreviations-and-symbols/",
+			`Glossary of symbols and abbreviations, including Latin, medication and dosage abbreviations used in the ${siteTitleShort} and prescribing.`
+		);
+		metas.set(
+			"/about/medicines-information-services/",
+			`Find contact details for both regional and national medicines information and advice lines/services in the UK.`
+		);
+		metas.set(
+			"/about/paediatric-formulary-committee/",
+			`The Paediatric Formulary Committee (PFC) is responsible for the content of ${siteTitleShort}. The PFC includes pharmacy, medical, nursing and lay representatives.`
+		);
+
+		return metas;
+	}, [siteTitleShort, siteTitleLong, isBNF]);
+
+	const metaDesc = metaDescriptions.get(pathname);
+
+	// Throwing an error here means we break the build if a new about page gets
+	// added to the feed but that's a) infrequent and b) what we want so we can
+	// craft a new meta description.
+	if (!metaDesc)
```

Comment:
Might be wrong, but I think we left a meta description intentionally blank in one of the writing sessions with Angus. If so, would that break the build here?

Reply:
Ahhhh I think that was because one of the pages (Paediatric Formulary Committee) is BNFC only. So it won't break the build here